timescale-docker/sleep.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+/usr/local/bin/docker-entrypoint.sh postgres || true
```

Comment:
Why ignore failure?

Reply:
This is just a simple and quick script which get’s the DB up and running.

There could be much more error handling, but to avoid scope creeping I have tried made is simple.